src/stories/tests/ComponentTests.stories.tsx

```diff
@@ -37,8 +42,14 @@ const defaultArgs: CioAutocompleteProps = {
       indexSectionName: 'Products',
     },
   ],
+  ...explicitActionsSpies,
 };
 
+// @ts-ignore
+// eslint-disable-next-line
+window.navigator.__defineGetter__('webdriver', () => false);
```

Comment:
Override `webdriver` so our tracker module doesn't treat it as a bot and block tracking events